packages/dashboard/src/customization/widgets/lineSymbol/lineAnchor/component.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { CSSProperties, useEffect, useRef } from 'react';
+import { LineWidget } from '~/customization/widgets/types';
+import { UpdateWidgetsAction } from '~/store/actions';
+import { XYCoord, useDrag } from 'react-dnd';
+import { DashboardWidget } from '~/types';
+import { useGridSettings } from '~/components/actions/useGridSettings';
+import './component.css';
+
+export const LineAnchor: React.FC<{
+  style: CSSProperties;
+  anchorType: 'start' | 'end';
+  widget: LineWidget;
```

Comment:
does the anchor need to know about the entire widget or can we pair this down to just a point and updatePoint?

Reply:
i think it does, but i'm not sure. how else would we `updateWidget`?